test/layer2libTest.js

```diff
@@ -4,7 +4,7 @@ const web3 = new Web3()
 const Layer2lib = require('../src/index.js')
 const Promise = require('bluebird');
 
-const redis = require('redis')
```

Comment:
Can you comment this out rather than delete as I will need to have this for my tests

Reply:
done!